deployment/cloud-run/secure_api_server.py

```
@@ -479,34 +530,488 @@
+
+        result = {
+            'summary': summary,
+            'original_length': original_length,
+            'summary_length': summary_length,
+            'compression_ratio': compression_ratio,
+            'processing_time': time.time() - start_time
+        }
+        logger.info(f"📤 Summarization result: {result}")
+        return jsonify(result)
+
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)